packages/reports/src/Components/Routes/__tests__/binary-routes.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { createBrowserHistory } from 'history';
+import { render, screen } from '@testing-library/react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import BinaryRoutes from '../binary-routes';
+
+jest.mock('@deriv/translations', () => ({
+    Localize: jest.fn(({ i18n_default_text }) => <div>{i18n_default_text}</div>),
+    localize: jest.fn(text => text),
+}));
+
+jest.mock('../route-with-sub-routes', () => jest.fn(() => <div>RouteWithSubRoutes</div>));
+
+jest.mock('Constants/routes-config', () => () => [{}]);
+
+describe('<BinaryRoutes />', () => {
+    const history = createBrowserHistory();
+
+    it('should render BinaryRoutes with mocked route component', () => {
+        const mock = mockStore({
+            modules: {
+                common: {
+                    current_language: 'EN',
```

Comment:
`EN` might be the default language, will it work if you just leave `const mock = mockStore({});`?
<img width="359" alt="Screenshot 2024-06-24 at 8 46 59 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/273e869f-536f-44af-a564-99fea08d3ba1">